plugins/file_logs.yaml

```diff
@@ -44,11 +44,11 @@ parameters:
     type: string
     default: "file"
   - name: include_file_name
-    description: Whether to add the file name as the attribute log.file.name.
+    description: Whether to add the file name as the body log.file.name.
```

Comment:
These descriptions can stay the same; These should stay as attributes (they currently do, too).

Reply:
updated back to original